includes/definitions/discovery/innovaphone.yaml

```diff
@@ -0,0 +1,64 @@
+mib: INNO-MIB
+modules:
+    sensors:
+        state:
+            data:
+                -
```

Comment:
This YAML state definition looks wrong . You walk a table but do not use the index you get and instead hardcode the index. That cannot give the proper result (at least not every time). Like below, you hardcode PRI1-L2 as the index, and may display the wrong value for another index.